File: problemas/ad-hoc/2342/2342.jl
using Printf

n = parse(Int, readline())

palavras = split(readline())
p = parse(Int, palavras[1])
c = palavras[2]
q = parse(Int, palavras[3])

if c == "+"
  if p + q <= n
    @printf "OK\n"
  else
    @printf "OVERFLOW\n"
  end
else
  if p * q <= n
    @printf "OK\n"
  else
    @printf "OVERFLOW\n"
  end
end

File: problemas/ad-hoc/2342/2342.cs
using System;

class URI {
    static void Main(string[] args) {
        string C;
        int N, P, Q;

        N = int.Parse(Console.ReadLine());
        string[] entrada = Console.ReadLine().Split(' ');

        P = int.Parse(entrada[0]);
        C = entrada[1];
        Q = int.Parse(entrada[2]);

        if(C == "+"){
            if(P + Q <= N)
                Console.WriteLine("OK");
            else
                Console.WriteLine("OVERFLOW");
        }else{
            if(P * Q <= N)
                Console.WriteLine("OK");
            else
                Console.WriteLine("OVERFLOW");
        }
    }
}